src/modules/ModulesRouter.tsx

```diff
@@ -9,13 +9,15 @@ const Home = withLazy(() => import('./home'));
 const Login = withLazy(() => import('./login'));
 const Main = withLazy(() => import('./main'));
 const Register = withLazy(() => import('./register'));
+const Admin = withLazy(() => import('./admin'));
```

Comment:
Admin should be added inside Main module not core - because Admin uses Layout from Main and also we have option to use this module only in application